WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModel.kt

```diff
@@ -233,21 +243,30 @@ class SiteCreationDomainsViewModel @Inject constructor(
 
     private fun createSuggestionsUiStates(
         onRetry: () -> Unit,
+        query: String?,
         data: List<String>,
         errorFetchingSuggestions: Boolean,
         @StringRes errorResId: Int?
     ): List<DomainsListItemUiState> {
         val items: ArrayList<DomainsListItemUiState> = ArrayList()
         if (errorFetchingSuggestions) {
             val errorUiState = DomainsFetchSuggestionsErrorUiState(
-                    messageResId = errorResId ?: R.string.site_creation_fetch_suggestions_error_unknown,
+                    messageResId = errorResId
+                            ?: R.string.site_creation_fetch_suggestions_error_unknown,
```

Comment:
NitPick: Could you please revert this change. Having this an a new line seems weird.

Reply:
Yes, will revert it 🙏